exo-run/features/support/world.ls

```diff
@@ -24,4 +29,5 @@ World = !->
 
 
 module.exports = ->
+  @app-dir = app-dir
```

Comment:
the `@app-dir` variable has to be used by multiple of the `given/when/then` steps, so it needs to have the same value when being called from the different files.

The comment associated with that variable in `world.ls` does seem misleading in regards to it only being used for end-to-end tests. It was copied over from the original `steps.ls` files, but I think it should be removed?